factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/registry/transformation/chains/TransformationChains.java

```diff
@@ -56,7 +71,14 @@ public static Edge from(Transformation t) {
         }
     }
 
-    public TransformationChain build(TransformationKey key, int from, int to)
+    public TransformationChain get(TransformationKey key, int from, int to)
```

Comment:
Do we need some kind of synchronization here? 🤔

Reply:
that would potentially block a lot of people unnecessarily, and it is in the critical path